examples/google-secure-signals-integration/with_sdk_v3/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uid2-publisher",
-  "version": "1.0.0-28f607bff9",
+  "version": "1.0.0",
```

Comment:
If we have already built a version 1.0.0, should this not be 1.0.1?

Reply:
It was a failed patch run that's why I kept it as 1.0.0. Should we still bump it to 1.0.1?